src/css/base.scss

```diff
@@ -159,4 +161,55 @@ h3 {
 
 .hidden {
   display: none;
+}
+
+// ~~~~~~~
+.login-form {
+  // border: 1px solid red;
+  height: 50vh;
+  width: 55vh;
+  box-shadow: 0 10px 14px rgb(34 15 15 / 13%), 0 4px 14px rgb(0 0 0 / 22%);
+  // margin-top: 100px;
+  display: flex;
```

Comment:
Opportunity to add a mixin for display flex, flex direction column, etc.  I also don't see any SASS variables currently in use. Lets discuss.